com.blitterstudio.amiberry.yml

```diff
@@ -0,0 +1,98 @@
+app-id: com.blitterstudio.amiberry
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: amiberry.sh
+rename-icon: amiberry
+copy-icon: true
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  # Wayland access
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Needs to talk to the network:
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: libserialport
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: archive
+        url: https://github.com/sigrokproject/libserialport/archive/refs/tags/libserialport-0.1.1.tar.gz
+        sha256: 4e3573e3bdbb6b0a872f32c4826d1b6b630dcc3eeb5f05d0501bb79d738a09ea
+
+  - name: amiberry
+    only-arches:
+      - x86_64
+    post-install:
+      - install -Dp -m 755 amiberry /app/bin/amiberry
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    builddir: true
+    
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/archive/refs/tags/v5.6.2.tar.gz
+        sha256: 4e9feb3ab0039871a25dda9f42a29c9ec1d6160ff68ef7c37fd3205f59c14b97
+
+  - name: desktop
+    buildsystem: simple
+    build-commands:
+      - install -D -m644 -t /app/share/applications/ ${FLATPAK_ID}.desktop
+      - install -D -m644 -t /app/share/metainfo/ ${FLATPAK_ID}.metainfo.xml
+      - install -D -m755 -t /app/bin/ amiberry.sh
+      - install -Dm644 256x256.png /app/share/icons/hicolor/256x256/apps/amiberry.png
+      - mkdir -p /app/data
+      - cp -R abr /app/data
+      - cp -R conf /app/data
+      - cp -R controllers /app/data
+      - cp -R data /app/data
+      - cp -R inputrecordings /app/data
+      - cp -R kickstarts /app/data
+      - cp -R nvram /app/data
+      - cp -R savestates /app/data
+      - cp -R screenshots /app/data
+      - cp -R whdboot /app/data
```

Comment:
where do these come from?

Reply:
they are part of the Amiberry sources, contained in that repo (and tarball). Would it work if we merge these two modules into one, since we're getting the tarball and building the app in the previous step?

Reply:
there is no amiberry source here.

Reply:
obviously you never built this locally.

Reply:
I built it, but since I had the amiberry sources together I didn't run into this problem. The manifest and the rest of the files are included in the Amiberry repo, and that's how I've been testing the flatpak so far. I'll make the required changes to fetch everything from upstream then, thanks!

Reply:
Tested to build locally now, only from the manifest :)